hw/cdc/tools/verixcdc/run-cdc.tcl

```diff
@@ -1,237 +0,0 @@
-
-# Used to disable some SDC constructs that are not needed by CDC.
-set IS_CDC_RUN 1
-
-########################
-## Library Setup      ##
-########################
-
-# if the foundry root is specified, some more library setup is needed.
-if {$FOUNDRY_ROOT != ""} {
-  # TODO: add lib setup tcl file here
-  # this PRIM_DEFAULT_IMPL selects the appropriate technology by defining
-  # PRIM_DEFAULT_IMPL=prim_pkg::Impl<tech identifier>
-  # PRIM_DEFAULT_IMPL is set inside the library setup script
-  set DEFINE "PRIM_DEFAULT_IMPL=${PRIM_DEFAULT_IMPL}+${PRIM_STD_CELL_VARIANT}"
-  source "${FOUNDRY_ROOT}/cdc/verixcdc/setup.tcl"
-} else {
-  set DEFINE ""
-}
-
-########################
-## Configure CDC Tool ##
-########################
-
-# TODO: potentially more settings are needed.
-# set ri_enable_sva false
-set ri_create_outputs_in_create_env true
-set ri_print_module_nand2_counts true
-# enable analysis of large arrays
-set ri_max_total_range_bits 20
-set ri_prefer_liberty true
-
-############################
-## Apply Liberty File ##
-
-read_liberty $AST_LIB -db_dir ./RI_compiled_libs/verix_libs
-############################
-
-#########################
-## Analyze & Elaborate ##
-#########################
-
-if {$DEFINE != ""} {
-  analyze -sverilog +define+${DEFINE}  -f ${SV_FLIST}
-} else {
-  analyze -sverilog   -f ${SV_FLIST}
-}
-
-# TODO(#13197): two flops are accessing the same memory cells
-# prim_generic_ram_2p is not recognized as a CDC module by the tool
-# So, it's black-boxed while waiting for a tool patch
-if {$PARAMS != ""} {
-  elaborate -params "$PARAMS" $DUT -black_box prim_generic_ram_2p
-} else {
-  elaborate $DUT -black_box prim_generic_ram_2p
-}
-#################################
-## Define Common Synchronizers ##
-#################################
-
-# Glitch Free Mux
-# WARNING!!! prim_clock_mux2 is not a glitch free mux
-#set_user_glitch_free_muxes -name opentitan_clock_mux prim_generic_clock_mux2
-#set_user_glitch_free_muxes -name opentitan_glitchfree_mux prim_generic_clock_glitchfree_mux2
-
-# 2FF synchronizer.
-# TODO: Process dependent module name later
-set prim_2ff_modules {}
-
-# Find every derivated modules from 2FF synchronizer
-foreach mod [get_all_modules prim_flop_2sync] {
-  lappend prim_2ff_modules $mod
-  puts "Adding to list prim_2ff_modules: $mod"
-}
-
-set_user_cntl_synchronizer -name opentitan_2ff $prim_2ff_modules
-
-# Pulse synchronizer
-
-# Req/Ack synchronizer
-
-#########################
-## Apply Constraints   ##
-#########################
-
-#read_sdc $CONSTRAINT
-create_scenario {sdc env}
-current_scenario sdc
-read_sdc  $CONSTRAINT  -output_env constraints.sdc.env
-current_scenario env
-read_env constraints.sdc.env
-touch $USER_ENV_FILE
-read_env $USER_ENV_FILE
-
-# RI's recommended approach to filter out unnecessary errors on async_fifo
-catch {rm tmp.env}
-set outfile [open tmp.env  w]
-puts $outfile "set_data_clock_domain rdata_o -module prim_fifo_async -derived_from rvalid_o"
```

Comment:
Is this something we should do in Meridian as well?

Reply:
Yes, added